### PR TITLE
🐛fix: oak text editor font size - DASHV6-226

### PR DESCRIPTION
**Pull request by maximedasilva**
This fix editor base font size to avoid text increasing in editor to result in smaller text in component. 

In addition, I also added some translations for all the content editor tooltips.

### CODE REVIEW

File: packages/oak-addon-basic-components/lib/core/Editor/ColorButton.js [hunk 1]
```diff
@@ -56,7 +57,12 @@ export default ({ className }) => {
   return (
     <Dropdown className="oak-color-field">
       <DropdownToggle tag="span">
-        <Tooltip text="Color">
+        <Tooltip text={(
+          <Text>{t => t('addons.basicComponents.fields.editor.color',
```
Comment (dackmin) [inline packages/oak-addon-basic-components/lib/core/Editor/ColorButton.js:61]:
```jsx
<Text
  name="addons.basicComponents.fields.editor.color"
  default="Color"
/>
```

File: packages/oak-addon-basic-components/lib/core/Editor/index.js [hunk 2]
```diff
@@ -74,50 +99,86 @@ export default ({
           <MarkButton
             format="bold"
             icon="format_bold"
-            tooltipText="Bold"
+            tooltipText={(
+              <Text>{t => t('addons.basicComponents.fields.editor.bold',
```
Comment (dackmin) [inline packages/oak-addon-basic-components/lib/core/Editor/index.js:103]: use `name` & `default` props instead

File: packages/oak-addon-basic-components/lib/core/Editor/BlockButton.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import { useSlate } from 'slate-react';
 import { classNames } from '@poool/junipero-utils';
+import { Tooltip } from '@poool/junipero';
```
Comment (dackmin) [inline packages/oak-addon-basic-components/lib/core/Editor/BlockButton.js:3]: import classNames from `@poool/junipero` directly

**Review by dackmin — CHANGES_REQUESTED** [review]

File: packages/oak-addon-basic-components/lib/core/Editor/BlockButton.js [hunk 4]
```diff
@@ -12,19 +12,21 @@ export default ({ format, icon, className }) => {
...
+        className={classNames(
+          'oak-toolbar-button',
+          'oak-' + format,
+          {
+            'oak-active': isBlockActive(editor, format),
+          },
+          className,
+        )}
+      >
+        <i className="oak-icons">{ icon }</i>
```
Comment (dackmin) [inline packages/oak-addon-basic-components/lib/core/Editor/BlockButton.js:28]: use `<Icon>{ icon }</Icon>`

**Review by dackmin — CHANGES_REQUESTED** [review]

**Comment by maximedasilva** [comment]
I still have a problem with a traduction key (Bold to be precise), in english no text is rendered and I have this error 
Capture d’écran 2021-07-30 à 18 24 23

It seems to come from `junipero-utils/get` function

**Comment by dackmin** [comment]
The `.get` problem comes from poorly written code inside junipero-utils: https://github.com/p3ol/junipero/issues/598
I'll release a patch and update your PR once it is available on npm.

**Pull request merged**